### PR TITLE
Documentation of dicm2nii

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Documentation of the function dicm2nii. Description and INPUTS in need of completion.

### CODE REVIEW

File: ea_dicm2nii.m [hunk 1]
```diff
@@ -1,3 +1,17 @@
 function ea_dicm2nii(indir,outdir)
+%
```
Comment (Daniel-Tojal) [inline ea_dicm2nii.m]: Please provide a description of this function.

File: ea_dicm2nii.m [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+%
+% USAGE:
+%
+%    ea_dicm2nii(indir,outdir)
+%
+% INPUTS:
```
Comment (Daniel-Tojal) [inline ea_dicm2nii.m]: Please provide a description of the inputs.

Comment (ningfei) [reply ea_dicm2nii.m]:
```
%    indir: DICOM folder
%    outdir: output folder for NIfTI files
```

File: ea_dicm2nii.m (hunk 1 above)
Comment (ningfei) [reply ea_dicm2nii.m]: Wrapper for dicm2nii (Xiangrui Li, https://github.com/xiangruili/dicm2nii)

**Comment by laurentheirendt** [comment]
@Daniel-Tojal, the inputs need to be properly formatted.

**Comment by laurentheirendt** [comment]
Thanks @Daniel-Tojal and @ningfei 👍

**Pull request merged**